tests/cephadm/test_cephadm_operations_via_admin.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from cli.cephadm.cephadm import CephAdm
+
+
+class CephadmOperationsAdmin(Exception):
+    pass
+
+
+def run(ceph_cluster, **kw):
+    """Verify cephadm operations via admin node for managing and expanding cluster
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+        e.g
+        test:
+            name: test_cephadm_operations_via_admin_node
+            desc: Verify cephadm operations via admin node for managing and expanding cluster
+            config:
+                label: _admin
+            polarion-id: CEPH-83573725
+            module: test_cephadm_operations_via_admin.py
+    """
+    config = kw.get("config")
+    label = config.get("label")
```

Comment:
Add validation for these parameters as these are mandatory

Reply:
Thanks @vamahaja 
Updated